### PR TITLE
Include the new adapter board DS2 and remove machine openvario-7-AM070

### PR DESCRIPTION
**Pull request by Blaubart**
This PR based on the PR [118](https://github.com/Openvario/meta-openvario/pull/118). Sorry I make a new PR. I'm not so familiar with GitHub right now, and I deleted my fork accidentally.

### CODE REVIEW

File: conf/machine/openvario-57-lvds_DS2.conf [hunk 1]
```diff
@@ -43,7 +43,7 @@ MACHINE_FEATURES ?= "alsa apm keyboard rtc serial screen usbgadget usbhost vfat"
 
 SOC_FAMILY = "sun7i"
 
-KERNEL_DEVICETREE_SOURCE = "openvario-7-AM070.dts"
+KERNEL_DEVICETREE_SOURCE = "openvario-57-lvds_DS2.dts"
```
Comment (kedder) [inline conf/machine/openvario-57-lvds_DS2.conf:46]: The name is separated by dashes, let's keep it consistent: `openvario-57-lvds-DS2`. Don't mix underscores and dashes: makes it hard to type correctly and causes confusion.

File: recipes-bsp/u-boot/files/openvario-7-CH070_DS2/openvario_defconfig.bak [hunk 2]
```diff
@@ -0,0 +1,44 @@
+CONFIG_ARM=y
+CONFIG_ARCH_SUNXI=y
```
Comment (kedder) [inline recipes-bsp/u-boot/files/openvario-7-CH070_DS2/openvario_defconfig.bak:2]: Please remove the .bak file from the git repo.

File: recipes-kernel/linux/linux-mainline/openvario-57-lvds_DS2.dts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        hactive = <640>;
+        vactive = <480>;
+        hsync-len = <1>;
+        hfront-porch = <2>;
+        hback-porch = <157>;
+        vsync-len = <1>;
+        vfront-porch = <2>;
+        vback-porch = <42>;
+    };
+};
```
Comment (kedder) [inline recipes-kernel/linux/linux-mainline/openvario-57-lvds_DS2.dts:21]: Is this a complete copy of `openvario-57-lvds.dts`?

File: recipes-apps/ovmenu-ng/files/openvario-57-lvds_DS2/ovmenu-ng.sh [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+
+setfont default8x16
+
+DIALOG_CANCEL=1 dialog --nook --nocancel --pause "Starting XCSoar ... \\n Press [ESC] for menu" 10 30 $TIMEOUT 2>&1
+
+case $? in
+	0) start_xcsoar;;
+	*) main_menu;;
+esac
+main_menu
```
Comment (kedder) [inline recipes-apps/ovmenu-ng/files/openvario-57-lvds_DS2/ovmenu-ng.sh:396]: Did you change anything in this file or just copy-pasted it?

File: recipes-kernel/linux/linux-mainline/openvario-57-lvds_DS2.dts (hunk 3 above)
Comment (Blaubart) [reply recipes-kernel/linux/linux-mainline/openvario-57-lvds_DS2.dts:21]: No, UART configuration has changed.

File: recipes-apps/ovmenu-ng/files/openvario-57-lvds_DS2/ovmenu-ng.sh (hunk 4 above)
Comment (Blaubart) [reply recipes-apps/ovmenu-ng/files/openvario-57-lvds_DS2/ovmenu-ng.sh:396]: I copy-paste these files for the new machines. They are necessary for compiling and I don't know an other method

**Comment by Blaubart** [comment]
Hello Andrey, are my revisions OK? should I adjust something?

**Review by kedder — APPROVED** [review]
This looks good to me. All the duplication is not great, but I guess we can live with it for a while.

**Review by linuxianer99 — APPROVED** [review]

**Pull request merged**